### ISSUE TITLE
ENH: Add Functionality to extract embeddings

### ISSUE THREAD

**Issue by StatMixedML** [amazon-science/chronos-forecasting#26]
# Summary
This issue is a suggestion for adding functionality to Chronos that allows users to extract embeddings directly. This feature would greatly enhance the model's utility by enabling more detailed analysis and application in various forecasting tasks.

**Comment by abdulfatir** [comment]
@StatMixedML Thanks for opening. I am already working on this in #24.

**Comment by abdulfatir** [comment]
Done in #24.

**Comment by StatMixedML** [comment]
@abdulfatir Thanks for the quick release!

I am running the example as shown in the README.md. The data has shape `df.shape = (144, 2)`, whereas the embeddings have shape `embeddings.shape = torch.Size([1, 145, 768])`. Is there reason why the second dimension, i.e., the time dimension of the embeddings has one more observation than the original data?

**Comment by abdulfatir** [comment]
@StatMixedML it's described in the docstring: https://github.com/amazon-science/chronos-forecasting/blob/b4a6c0c2eb5af70853b17a8709a6ebf4e7945b23/src/chronos/chronos.py#L359

TL;DR: it's for the EOS (end of sequence) token.

**Comment by StatMixedML** [comment]
Ok thanks, so this means we should disregard the last embedding `embeddings[:,:-1,:] ` for downstream tasks?
